kudu/gulpfile.js

```diff
@@ -8,7 +8,7 @@ const autorest = require('autorest');
 const path = require('path');
 const fse = require('fs-extra');
 
-gulp.task('build', async () => {
+gulp.task('build', async (callback) => {
```

Comment:
Doesn't seem like you should mix models - either return a promise or use a callback, not both...

Reply:
on a timer??